src/04-modality-specific-files/05-task-events.md

```diff
@@ -102,20 +108,40 @@ onset duration  trial_type  identifier  database  response_time
 5.6 0.6 sad AF01ANSA  kdef  1.739
 ```
 
-This file should be accompanied by a data dictionary.  
+This file should be accompanied by a data dictionary.
 
 ```Text
 sub-control01/
     func/
         sub-control01_task-emoface_events.json
 ```
 
-The `trial_type` and `identifier` columns from the `*_events.tsv` files might be described in this
-dictionary as follows. 
-Note that all other columns SHOULD also be described but are omitted for the sake
-of example.
+Specific fields related to stimulus presentation details SHOULD also be present
+in this data dictionary:
+
+| Column name                      | Description                                                                                                                                                                                                                                  |
+|----------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| StimulusPresentationOS           | RECOMMENDED. Operating system used to run the stimuli presentation software (for formatting recommendations, see examples below this table).                                                                                                 |
+| StimulusPresentationSoftwareName | RECOMMENDED. Name of the software that was used to present the stimuli                                                                                                                                                                       |
+| StimulusPresentationSoftwareRRID | RECOMMENDED. Name ideally with its [Research Resource Identifier](https://scicrunch.org/resources) of the software that was used to present the stimuli Examples: the RRID for Psychtoolbox is 'SCR_002881' that of PsychoPy is 'SCR_006571' |
+| StimulusPresentationVersion      | RECOMMENDED. Version of the software that was used to present the stimuli.                                                                                                                                                                   |
+| StimulusPresentationCode         | RECOMMENDED. DOI of the archive (for example on Zenodo, figshare...) for the code that was used to present the stimuli. Alternatively this could be a URL to code on GitHub, GitLab, etc.                                                    |
+The operating system description should include:
```

Comment:
line break between table and following paragraph

```suggestion
| StimulusPresentationCode         | RECOMMENDED. DOI of the archive (for example on Zenodo, figshare...) for the code that was used to present the stimuli. Alternatively this could be a URL to code on GitHub, GitLab, etc.                                                    |

The operating system description should include:
```